docs/END_USER_DOCUMENT.md

```diff
@@ -750,3 +751,38 @@ AsyncBooksAppClient asyncClient = MoccaClient.Builder
         .client(executorClient)
         .build(AsyncBooksAppClient.class);
 ```
+## 8. Request Validation
+Mocca now supports validation of request parameters using a standard bean validation 2.0 implementation like hibernate.
```

Comment:
This text will probably last for many more releases still to come. Because of that, I believe we should remove the word "now" from here.

Reply:
sure